backend/src/models/game.model.ts

```diff
@@ -6,23 +6,20 @@ import { RoundSchema, Round } from "./round.model";
 
 export enum GameState {
   lobby = "LOBBY",
-  roundBefore = "ROUND_BEFORE",
-  roundPlayers = "ROUND_PLAYERS_CHOOSE",
-  roundHost = "ROUND_HOST_CHOOSE",
-  roundAfter = "ROUND_AFTER",
+  active = "ACTIVE",
   finished = "FINISHED",
 }
 
 export interface Game extends Document {
   gameCode: string;
   settings: Settings;
   setups: Types.DocumentArray<Setup>;
-  discardedSetups?: Types.DocumentArray<Setup>;
+  discardedSetups: Types.DocumentArray<Setup>;
   punchlines: string[];
-  discardedPunchlines?: string[];
+  discardedPunchlines: string[];
   players: Types.DocumentArray<Player>;
-  state?: GameState;
-  rounds?: Types.DocumentArray<Round>;
+  state: GameState;
+  rounds: Types.DocumentArray<Round>;
```

Comment:
Why are you getting rid of them being optional? Was it an oversight prior?

Reply:
Yeah this was a misunderstanding on how the Mongoose model typing works - optional in the interface means that field may be undefined in the DB which is not what we want - we just want some fields to be required to be set to initialise (such as gameCode, punchlines, setups) & others that default to an initial value